client/elements/static/home-page.html

```diff
@@ -439,7 +439,9 @@
      <div class="big-card-title" inner-text="{{localize('24cbc4d3253a2779ae4dab326e42659b')}}">
      </div>
      <div class="three-elements-grid">
-      <paper-card class="three-elements-big paper-card-with-button" elevation="2" image="/img/home-page/14.jpg">
+      <paper-card alt="Sixteen gilt and lacquer leaves with two gilt and lacquer book covers. Wood coated with lacquer, 
```

Comment:
hmm, this should have been extracted by the localizer extraction script. @gradam